### PR TITLE
Feature data objects

### PR DESCRIPTION
**Pull request by aviadud**

### CODE REVIEW

File: app/src/main/res/layout/activity_chat_test.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (izipris) [inline app/src/main/res/layout/activity_chat_test.xml]: do we need this test file?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,3 +1,8 @@
 <resources>
     <string name="app_name">Benefit</string>
+
+    <!-- Facebook application ID and custom URL scheme (app ID prefixed by 'fb'). -->
```
Comment (izipris) [inline app/src/main/res/values/strings.xml:4]: is it needed?

File: app/src/main/res/layout/chat_massage.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (izipris) [inline app/src/main/res/layout/chat_massage.xml]: do we need this file? looks like relevant for the chat that we dropped.

**Review by izipris — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_chat_test.xml (hunk 1 above)
Comment (aviadud) [reply app/src/main/res/layout/activity_chat_test.xml]: No!

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (aviadud) [reply app/src/main/res/values/strings.xml:4]: Yes - for facebook login

File: app/src/main/res/layout/chat_massage.xml (hunk 3 above)
Comment (aviadud) [reply app/src/main/res/layout/chat_massage.xml]: right!

**Review by izipris — APPROVED** [review]

**Pull request merged**